react-mobx/src/main/scala/demo/ObservingFC.scala

```diff
@@ -1,11 +0,0 @@
-package slinky.core
```

Comment:
I had forgot I had implemented this twice, and this is way shorter. should we keep this implementation and the other name?

Reply:
So should I use this leaner implementation also to substitute https://github.com/ScalablyTyped/mobx-slinky/blob/master/src/main/scala/slinky/core/ObserverComponent.scala?

Reply:
well yeah I suppose so. It's not very important though, they should work exactly the same